src/Arius.Benchmarks/raw/20260429T113931.866Z/results/Arius.Benchmarks.RepresentativeWorkflowBenchmarks-report-github.md

```
@@ -0,0 +1,15 @@
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)